### DISCUSSION TITLE
Adding packages/modules to existing conda environment

### DISCUSSION THREAD

**Discussion by drewmresnick** [NOAA-GFDL/MDTF-diagnostics#205] [Q&A] [answered]
The conda environment "_MDTF_python3_base" provides most of the packages/modules that are required for running my code. However there are two which are not. Should I include in my pull request a yaml file so that a new environment would be created, or could these two simply be added to a preexisting environment?

**Comment by wrongkindofdoctor** [comment, marked as answer]
@drewmresnick Can you add the modules that you need to this post? If other developers need them, or are likely to at some point, we can add them to the python3 base yaml.  For now, though, please maintain a separate yaml for your POD with the extra modules.

**Reply by drewmresnick** [reply]
Okay this sounds good. The modules are: sklearn (scikit-learn) and numba
